app/models/comment.rb

```diff
@@ -0,0 +1,24 @@
+class Comment < ApplicationRecord
+  # Association: This defines an association between the Comment model and the User & post model.
+  # It specifies that each comment belongs to an author (user) & post. The class_name option is
+  # used to specify the class name of the associated model (in this case, 'User' & 'Post').
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Attribute - This sets up an attribute for the Comment model called text, with the
+  # data type of :text. It's a way to explicitly define an attribute and its data type.
+  attribute :text, :text
+
+  # Callbacks: This callback is triggered after a new comment is created/destroyed(deleted).
+  # It calls the update_post_comments_counter method to update the comments counter for
+  # the associated post
+  after_create :update_post_comments_counter
+  after_destory :update_post_comments_counter
```

Comment:
* Kindly fix this error i get as a result of a typo in your comment model `after_destory` instead of `after_destroy` 
![image](https://github.com/unorjikingsley/ROR-BlogApp/assets/52497246/963e8e10-665a-4b3d-bc93-1b2bcb383545)